hosted-cluster-config-operator/controllers/clusteroperator/reconcile.go

```diff
@@ -233,6 +233,13 @@ var clusterOperators = []ClusterOperatorInfo{
 			},
 		},
 	},
+	{
+		Name: "operator-lifecycle-manager-packageserver",
+		VersionMapping: map[string]string{
+			"operator": "release",
+		},
+		RelatedObjects: []configv1.ObjectReference{},
+	},
```

Comment:
@sjenning this doesn't seem to have been sufficient to get this clusteroperator resource created on the guest.  Did i miss a step?